tycho-its/src/test/java/org/eclipse/tycho/test/compiler/CompilerExtraExportsJava17.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat Inc. and others.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.tycho.test.compiler;
+
+import org.apache.maven.it.Verifier;
+import org.eclipse.tycho.test.AbstractTychoIntegrationTest;
+import org.junit.Test;
+
+public class CompilerExtraExportsJava17 extends AbstractTychoIntegrationTest {
+
+    @Test
+    public void testExtraExports() throws Exception {
+        Verifier verifier = getVerifier("compiler.extraExports.java17", false);
+        verifier.executeGoal("compile");
```

Comment:
You probably want to verifier.verifyErrorFreeLog() assertion here...

Reply:
Bit I want this in the other test case too?

Reply:
I always add this as it does not harm to assert there are no (other) errors beside a build-failure.

Reply:
I take this as a yes for the other test case (the non Java 17 one).